ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -73,12 +75,22 @@ private async Task SetTbServiceAndHospitalListsAsync()
 
             var hospitals = await _referenceDataRepository.GetHospitalsByTbServiceCodesAsync(tbServiceCodes);
             Hospitals = new SelectList(hospitals, nameof(Hospital.HospitalId), nameof(Hospital.Name));
-        }
 
-        public async Task<JsonResult> OnGetHospitalsByTbService(string tbServiceCode)
-        {
-            var tbServices = await _referenceDataRepository.GetHospitalsByTbServiceCodesAsync(new List<string> { tbServiceCode });
-            return new JsonResult(tbServices);
+            var caseManagers = await _referenceDataRepository.GetCaseManagersByTbServiceCodesAsync(tbServiceCodes);
+
+            // If case manager not yet set, and current user is an allowed case manager for
```

Comment:
I'd have thought we do this on notification creation, otheriwse it has that weird behavioru of being null but looking populated every time you look at the record?

Reply:
It would then also be logically grouped together with the bit where we set the service